src/main/java/org/gridsuite/study/server/service/StudyService.java

```diff
@@ -1332,9 +1337,11 @@ public String duplicateModifications(UUID studyUuid, UUID nodeUuid, List<UUID> m
         String response;
         try {
             checkStudyContainsNode(studyUuid, nodeUuid);
-            UUID targetGroupUuid = networkModificationTreeService.getModificationGroupUuid(nodeUuid);
-            response = networkModificationService.duplicateModification(targetGroupUuid, modificationUuidList);
-            updateStatuses(studyUuid, nodeUuid, false);
+            NodeModificationInfos nodeInfos = networkModificationTreeService.getNodeModificationInfos(nodeUuid);
+            UUID networkUuid = networkStoreService.getNetworkUuid(studyUuid);
+            response = networkModificationService.duplicateModification(modificationUuidList, networkUuid, nodeInfos);
+            // no invalidation cause we will apply the duplicated modifications on a built node
```

Comment:
Copy/Paste of a modification inside the same node should invalidate the children nodes of this node

Reply:
yes, I missed this point!

Reply:
this is also true for some cut/paste (when we have to build, between 2 unrelated subtrees)